Gems/ROS2/Code/Source/Lidar/ROS2Lidar2DSensorComponent.cpp

```diff
@@ -17,19 +17,19 @@
 
 namespace ROS2
 {
-    namespace Internal
+    namespace
     {
         const char* kLaserScanType = "sensor_msgs::msg::LaserScan";
     }
 
     void ROS2Lidar2DSensorComponent::Reflect(AZ::ReflectContext* context)
     {
-        Lidar2DSensorConfiguration::Reflect(context);
+        LidarSensorConfiguration::Reflect(context);
 
         if (auto* serialize = azrtti_cast<AZ::SerializeContext*>(context))
         {
             serialize->Class<ROS2Lidar2DSensorComponent, ROS2SensorComponent>()->Version(2)->Field(
-                "lidarConfiguration", &ROS2Lidar2DSensorComponent::m_lidarConfiguration);
+                "lidarBase", &ROS2Lidar2DSensorComponent::m_lidarBase);
```

Comment:
This will fail if ROS2LidarSensorComponent (the other one, unrelated) is not reflected, since LidarBase is not reflected then.